python/cuml/cuml/_thirdparty/sklearn/preprocessing/_imputation.py

```diff
@@ -13,17 +13,16 @@
 import numbers
 import warnings
 
+import cupy as np
+import numpy as cpu_np
```

Comment:
It would be nice to rewrite this (in a later PR) so that `np` really means NumPy and we use a different name like `cp` or `cupy` for CuPy. I'm not a fan of doing `import cupy as np` in library code. It is okay in notebooks where a user wants zero-code-change-like behavior.

If this import naming is done to reduce the diff in vendored code from sklearn, that might be an okay justification for keeping this as-is...

Reply:
I agree with you (I was also confused when I found these files). I suspect minimizing the diff was the original intent. Maybe we'll want to change these later, but not in this PR.